array-manipulation-3/src/unique.js

```diff
@@ -0,0 +1,28 @@
+/* exported unique */
+
+// store new empty array
+// put guard for empty input (if input is empty array, return [])
+// check each item in OG array
+//     check each item in new array
+//         if current OG item is eual to an item on new array
+//             break
+//         else push current OG item to new array
+// return new array
+
+function unique(array) {
+  if (array.length === 0) { return []; }
+  var Unique = [array[0]];
```

Comment:
You should avoid capital letters at the beginning of variables, that sort of casing is meant to be preserved for Constructor Functions